ckanext/fluent/validators.py

```diff
@@ -155,9 +156,13 @@ def validator(key, data, errors, context):
         if output is None:
             return
 
-        for lang in output:
-            del extras[prefix + lang]
-        data[key] = json.dumps(output, ensure_ascii=False)
+        # Return None if alle values in json are None or "" (state = deleted in package_extra)
+        if all(v is None or v == "" for v in output.values()):
+            data[key] = None
```

Comment:
Kom hij ook in de IF statement nr.3 terecht als de field_name de extentie _translated krijgt? Ik heb de indruk dat de value die wordt gevalideerd een multiLang dict en/of Json MultiLang dict is en dan komt hij in de IF statement nr1-2 terecht.

En daarin staat de data[key] nog niet in gedefinieerd. Zie comment boven

Reply:
Zowel notes_translated als qual_ass gaat naar nr. 3. Ze zijn beiden een missing object dus door de if-statement gaat die meteen naar nr 3. Nog te onderzoeken waarom die als een missing object ontvangen worden als input.